src/core/util/xlio_stats.h

```diff
@@ -351,9 +351,9 @@ typedef struct {
     cq_stats_t cq_stats;
 } cq_instance_block_t;
 
-typedef enum { RING_ETH = 0, RING_TAP } ring_type_t;
+typedef enum { RING_ETH = 0, RING_TAP, RING_ETH_TX } ring_type_t;
```

Comment:
order ETH, ETH_TX, TAP looks more readable (if you change order here, change it for the strings array too)

Reply:
Done